.readthedocs.yaml

```diff
@@ -13,7 +13,7 @@ build:
   jobs:
     post_checkout:
       - cd 'docs' && python3 build_requirements.py
-
+      
```

Comment:
question: Can this be dropped?

Reply:
Yes, sorry I didn't notice the whitespace.